packages/zowe-explorer/src/job/actions.ts

```diff
@@ -285,12 +285,36 @@ export async function deleteCommand(job: IZoweJobTreeNode, jobsProvider: IZoweTr
         return;
     }
 
-    // delete selected nodes
-    if (nodes.length > 0) {
-        for (const node of nodes) {
-            await jobsProvider.delete(node);
-            deletedNodes.push(`${node.job.jobname}(${node.job.jobid})`);
-        }
+    // delete selected multiple nodes
+    if (nodes.length > 1) {
+        await vscode.window.withProgress(
+            {
+                location: vscode.ProgressLocation.Notification,
+                title: localize("deleteJobPrompt.deleteCounter", "Deleting nodes"),
+                cancellable: true,
+            },
+            async (progress, token) => {
+                const total = 100;
+                for (const [index, currNode] of nodes.entries()) {
+                    if (token.isCancellationRequested) {
+                        vscode.window.showInformationMessage(
+                            localize("deleteJobPrompt.deleteCancelled", "Delete action was cancelled.")
+                        );
+                        return;
+                    }
+                    progress.report({
+                        message: `Deleting ${index + 1} of ${nodes.length}`,
+                        increment: total / nodes.length,
+                    });
+                    try {
+                        await jobsProvider.delete(currNode);
+                        deletedNodes.push(`${currNode.job.jobname}(${currNode.job.jobid})`);
+                    } catch (err) {
+                        // do nothing; delete next
```

Comment:
Is it possible to show the error and still continue to delete the next one?

Reply:
I think so, I just kept in consistent with the other similar command - multiple ds deletion.

Reply:
Yes, I am fine with it. Should I go with a warning message?

Reply:
yup. warning message will be good 👍 Thank you

Reply:
You can write it in the zowe/imperative log. I think we do it like this `this.log.error(error);`